src/analyze/analyze-security.h

```diff
@@ -1,7 +1,85 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
+#include <stdbool.h>
+
 #include "sd-bus.h"
+#include "hashmap.h"
+#include "format-table.h"
+
+typedef struct security_info security_info;
```

Comment:
you can inline this in the struct definition - `typedef struct security_info { ... } security_info;`

Reply:
if you do this switch to CamelCase please